coderd/database/migrations/000262_improve_notification_templates.up.sql

```diff
@@ -0,0 +1,128 @@
+-- https://github.com/coder/coder/issues/14893
+
+-- UserAccountSuspended
+UPDATE notification_templates
+SET
+	body_template = E'Hi {{.UserName}},\n\n' || -- Add a \n
```

Comment:
Awesome, thanks for including the change description.

Reply:
Well that's embarrassing... I might try letting a PR rest overnight so I can double check with fresh eyes before requesting review next time. Thanks for catching.